ckanext/switzerland/logic.py

```diff
@@ -100,3 +102,29 @@ def ogdch_dataset_by_identifier(context, data_dict):
         return result['results'][0]
     except (KeyError, IndexError, TypeError):
         raise NotFound
+
+
+@side_effect_free
+def ogdch_autosuggest(context, data_dict):
+    q = get_or_bust(data_dict, 'q')
+    lang = get_or_bust(data_dict, 'lang')
+    if lang not in ['en', 'it', 'de', 'fr']:
```

Comment:
Not so sure if this could be confusing to the user to only provide autosuggest if a language is chosen (even though this shouldn't be the case too often for a regular user).
Probably it is better this way, let the language be chosen and set "elsewhere" while we just deal with "the given" here.

Reply:
Not sure I understand this comment. But because we have language-specifzc suggestions only, this is currently the only way it works. Here "lang" is required, therefore a user of this API would get an error, if the parameter is missing.